Rubberduck.RibbonDispatcher/Sample Ribbons/SampleRibbon/src/RibbonModel.cls

```diff
@@ -0,0 +1,131 @@
+Attribute mCustomOptions.VB_VarHelpID = -1
+
+Private WithEvents mBrandingButton      As RibbonButton
+Attribute mBrandingButton.VB_VarHelpID = -1
+Private WithEvents mStandardButton1     As RibbonButton
+Attribute mStandardButton1.VB_VarHelpID = -1
+Private WithEvents mStandardButton2     As RibbonButton
+Attribute mStandardButton2.VB_VarHelpID = -1
+Private WithEvents mStandardOptions     As RibbonDropDown
+Attribute mStandardOptions.VB_VarHelpID = -1
+Private WithEvents mShowAdvancedToggle  As RibbonToggleButton
+Attribute mShowAdvancedToggle.VB_VarHelpID = -1
+
+Public Function Initialize(ByVal Factory As IRibbonFactory) As RibbonModel
+    On Error GoTo EH
+    Set this.BrandingGroup = Factory.NewRibbonGroup("BrandingGroup")
+    Set mBrandingButton = Factory.NewRibbonButtonMso("BrandingButton", , , , "MacroSecurity", True)
+    
+    Set this.CustomGroup = Factory.NewRibbonGroup("CustomButtonsGroup", False)
+    Set mCustomButton1 = Factory.NewRibbonButtonMso("AppLaunchButton1", True, True, rdLarge, "RefreshAll", True, True)
+    Set mCustomButton2 = Factory.NewRibbonButtonMso("AppLaunchButton2", True, True, rdLarge, "Refresh", True, True)
+    Set mCustomButton3 = Factory.NewRibbonButtonMso("AppLaunchButton3", True, True, rdLarge, "MacroPlay", True, True)
+    Set mCustomSizeToggle = Factory.NewRibbonToggleMso("SizeToggle", True, True, rdLarge, DefaultImage, False, True)
+    Set mCustomOptions = Factory.NewRibbonDropDown("ButtonOptions2", True, False)
+    mCustomOptions.AddItem Factory.NewSelectableItem("LabelOnly")
+    mCustomOptions.AddItem Factory.NewSelectableItem("ImageOnly")
+    mCustomOptions.AddItem Factory.NewSelectableItem("LabelAndImage")
+    
+    Set this.StandardGroup = Factory.NewRibbonGroup("StandardButtonsGroup")
+    Set mStandardButton1 = Factory.NewRibbonButtonMso("StandardButton1", True, True, rdRegular, "RefreshAll")
+    Set mStandardButton2 = Factory.NewRibbonButtonMso("StandardButton2", True, True, rdRegular, "Refresh")
+    Set mStandardOptions = Factory.NewRibbonDropDown("ButtonOptions", True, True)
+    Set mShowAdvancedToggle = Factory.NewRibbonToggle("ShowAdvancedToggle", True, True)
+    
+    mShowAdvancedToggle.OnToggled False
+    mStandardOptions.OnActionDropDown "", 2
+
+    mCustomOptions.SelectedItemId = "LabelAndImage"
+    mCustomOptions.IsEnabled = mCustomSizeToggle.IsPressed
+    
+    Set Initialize = Me
+XT: Exit Function
+EH: ReraiseError Err, ModuleName & "Initialize"
+    Resume XT
+    Resume Next
+End Function
+
+Private Sub mBrandingButton_Clicked()
+    MsgBox "Quack!", vbOKOnly Or vbInformation, "Rubberduck Ribbon Refresh"
+End Sub
+
+Private Sub mCustomButton1_Clicked()
+    DefaultButtonAction mCustomButton1
+End Sub
+Private Sub mCustomButton2_Clicked()
+    DefaultButtonAction mCustomButton2
+End Sub
+Private Sub mCustomButton3_Clicked()
+    DefaultButtonAction mCustomButton3
+End Sub
+Private Sub mCustomOptions_SelectionMade(ByVal ItemId As String, ByVal ItemIndex As Long)
```

Comment:
Whitespace between members

Reply:
Not following here.

Reply:
Both :smile: 

There should be a blank line between each member.